src/conf/local-file.yaml

```diff
@@ -1,7 +1,7 @@
 application:
   name: knoxautopolicy
   network:
-    operation-mode: 1                            # 1: cronjob | 2: one-time-job
+    operation-mode: 0                            # 1: cronjob | 2: one-time-job
```

Comment:
I think you added this for testing, can you please revert it back?

Reply:
Reverted